libs/designer/src/lib/ui/panel/panelroot.tsx

```diff
@@ -211,7 +211,7 @@ export const PanelRoot = (): JSX.Element => {
       panelLocation={PanelLocation.Right}
       isCollapsed={collapsed}
       noNodeSelected={!selectedNode}
-      isLoading={!iconUri}
+      // isLoading={!iconUri}
```

Comment:
What is this removing?

Reply:
It was for the loading icon in the operation panel header, I just pushed another commit to also check if it's a branching node so this is also fixed now